src/net_processing.cpp

```diff
@@ -2962,8 +2962,7 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
 
         // Store the new addresses
         std::vector<CAddress> vAddrOk;
-        int64_t nNow = GetAdjustedTime();
-        int64_t nSince = nNow - 10 * 60;
+        const std::chrono::seconds adjusted_current_time{GetAdjustedTime()};
```

Comment:
If we use `current_time` as variable name here (like above), it'll save on another round of renaming when use of adjusted times throughout address relay is removed.

Reply:
It is already named that way, see two lines below (2969)

Reply:
OK